samples/outlook/48-sensitivity-labels/sensitivity-labels-catalog.yaml

```
@@ -0,0 +1,80 @@
+id: outlook-sensitivity-labels-sensitivity-labels-catalog
+name: Sensitivity Labels Catalog
+description: This sample shows how to determine if sensitivity labels are enabled and to retrieve the catalog.
+host: OUTLOOK
+api_set: null
+script:
+    content: |
+        $("#get-sensitivity-labels-enabled").click(getSensitivityLabelsCatalogIsEnabled);
+        $("#get-sensitivity-labels-catalog").click(getSensitivityLabelsCatalog);
+
+        function getSensitivityLabelsCatalogIsEnabled() {
+          Office.context.sensitivityLabelsCatalog.getIsEnabledAsync(function(asyncResult) {
```

```suggestion
          Office.context.sensitivityLabelsCatalog.getIsEnabledAsync((asyncResult) => {
```
For consistency in docs and samples.